### DISCUSSION TITLE
Update issue - flex.symfony.com not found

### DISCUSSION THREAD

**Discussion by revil-O** [kimai/kimai#3758] [On-Premise: Installation & Updates] [answered]
the following error occurs on updating from version 1.14.x to 1.30.2

`curl error 6 while downloading https://flex.symfony.com/versions.json: Could not resolve host: flex.symfony.com `

to fix this error during installation/update to version 1.30.2 , run :

composer update symfony/flex --no-plugins --no-scripts

> 
  - Upgrading symfony/flex (v1.1x.x => v1.19.4)
>

hope this helps anyone who's facing this behavior.
--> regarding to this , please visit the following link: https://symfony.com/blog/the-old-flex-infrastructure-is-shutting-down

Kind regards

**Comment by kevinpapst** [comment, marked as answer]
Your solution is working, thank you very much for sharing 👍 

But people will likely run into git issues afterwards, as it will change composer.lock. And then they come back to me to ask for help^^

So I slightly adapted it and added it to the "update troubleshooting": 
https://www.kimai.org/documentation/updates.html#flexsymfonycom-not-found
